### PR TITLE
Fix static ip bug

### PR DESCRIPTION
**Pull request by JunqiZhang0**
Fixed the multiple network bug and made some changes on the static-DHCP-ip feature

### CODE REVIEW

File: linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml [hunk 1]
```diff
@@ -389,11 +392,8 @@
   loop: "{{ networks|product(res_count)|list }}"
 
 - set_fact:
-    macs: []
-
-- set_fact:
-    macs: "{{ macs + [mac_addresses.results[node.1].matches[node.0].mac.address] }}"
-  loop: "{{ range(0,networks|length,1)|list|product(res_count)|list }}"
+    macs: "{{ macs + [mac_addresses.results[node].matches[0].mac.address] }}"
+  loop: '{{ range(0,networks|length,1)|list }}'
```
Comment (abraverm) [inline linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml]: What about multiple systems (count > 1)?

File: linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml [hunk 2]
```diff
@@ -410,6 +410,31 @@
...
+    ip_set: []
+
+- set_fact: 
+    ip_set: "{{ip_set + [extract_ip_address_result.results[num]['stdout']|omit_filter(omit)]}}"
+  loop: "{{ range(0,networks|length, 1)|list}}"
+  loop_control:
+    loop_var: num
+  when: uri_hostname == 'localhost'
+
+# - debug:
```
Comment (abraverm) [inline linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml]: remove commented code

File: linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml [hunk 3]
```diff
@@ -410,6 +410,31 @@
   delay: 10
   when: uri_hostname == 'localhost'
 
+# ----------------------------------------------------------
```
Comment (abraverm) [inline linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml]: remove commented code

**Comment by samvarankashyap** [comment]
[test]

**Comment by samvarankashyap** [comment]
[test]

**Comment by JunqiZhang0** [comment]
@abraverm I didn't change any part of the code and ran the test locally it was passed. I don't know why the test on jenkins just magically failed. I'll try to rebase my code and submit a new pr today. That may save more time compared to debugging.

File: setup.py [hunk 4]
```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env python
 
 from __future__ import absolute_import
-import os
+
```
Comment (abraverm) [inline setup.py]: Why this file was changed?

**Comment by abraverm** [comment]
Looking at the commit history, it seems you have things mixed up. I suggest you clean up and submit one commit with all the needed changes.

**Comment by 14rcole** [comment]
@JunqiZhang0 The inventory generation in the `libvirt/01_template_inventory` integration test is failing: http://pastebin.test.redhat.com/782398

**Pull request closed without merging**